packages/sources/src/addSourceWizard/SSLFormLabel.js

```diff
@@ -1,23 +1,29 @@
 import React from 'react';
 import { QuestionCircleIcon } from '@patternfly/react-icons';
 import { Popover, Text, TextContent, TextVariants } from '@patternfly/react-core';
+import { FormattedMessage } from 'react-intl';
 
 const SSLFormLabel = () => (
     <React.Fragment>
-        SSL Certificate&nbsp;
+        <FormattedMessage id="wizard.SslCertificate" defaultMessage="SSL Certificate" />&nbsp;
         <Popover
             aria-label="Help text"
             maxWidth="50%"
             bodyContent={
                 <TextContent>
                     <Text component={ TextVariants.p }>
-                            You can obtain your OpenShift Container Platform provider’s CA
-                            certificate for all endpoints (default, metrics, alerts) from
-                        <b>/etc/origin/master/ca.crt</b>.
+                        <FormattedMessage
+                            id="wizard.YouCanObtainYourOpenshiftContainerPlatformProviderSCaCertificateForAllEndpointsDefaultMetricsAlertsFrom"
```

Comment:
I would definitely make these ids a bit more friendly. Don't forget that eventually, humans will have to go through the JSON and translate it. I would not want to read stuff like this.

Reply:
This can be changed later. Right now there is no common format and no one is translating it, so it doesn't matter. I don't want to come up with 207+ unique ids and scrap them later. :D

Reply:
It will be up to us to make the ids. Nobody (i hope:crossed_fingers:) can dictate stuff like that. But we can change it later.